packages/mui-material/src/Select/Select.js

```diff
@@ -221,6 +223,12 @@ Select.propTypes /* remove-proptypes */ = {
    * @default false
    */
   native: PropTypes.bool,
+  /**
+   * If `true`, multiple selection is available only when ctrl key is pressed.
+   * Only applicable when `multiple` is `true` and `native` is `false`.
+   * @default false
+   */
+  nativeLike: PropTypes.bool,
```

Comment:
If we plan to land this feature we need to come up with a better name for the prop, is too close to the native prop, and moreover works when the native prop is false. btw @atomiks do we have this feature planned for the Base UI select?

Reply:
Multi-select is planned for later

Reply:
@atomiks how much later? Asking so we can plan the Select refactor accordingly.

As for this feature, we should only implement it if we're sure that Base UI will eventually implement it. Otherwise, we would support it, only for it to change in the upcoming major version which is built on top of Base UI.